lib/github/github.ex

```diff
@@ -5,6 +5,33 @@ defmodule BorsNG.GitHub do
   Wrappers around the GitHub REST API.
   """
 
+  @doc """
+  Generic retry wrapper for GenServer calls that may timeout.
+  Retries up to 3 times with exponential backoff.
+  """
+  @spec retry_genserver_call(term, integer) :: term
+  defp retry_genserver_call(fun, retries \\ 3) when retries > 0 do
+    try do
+      fun.()
+    catch
+      :exit, {:timeout, _} ->
+        Logger.warn("GitHub API timeout, retrying (#{4 - retries}/3)")
+        # Exponential backoff: 1s, 2s, 3s
```

Comment:
Doesn't look exponential to me 😛 But it's no big deal.

Reply:
Agree.